sources/sections-a1-en/18-16th-cgpm.adoc

```diff
@@ -64,13 +64,13 @@ The 16th Conférence Générale des Poids et Mesures (CGPM),
 
 *recognizing* the general principles adopted for writing the unit symbols in Resolution 7 of the 9th CGPM (1948),
 
-*considering* that the symbol stem:["unitsml(l)"] for the unit litre was adopted by the Comité International des Poids et Mesures (CIPM) in 1879 and confirmed in the same Resolution of 1948,
+*considering* that the symbol stem:[sf "l"] for the unit litre was adopted by the Comité International des Poids et Mesures (CIPM) in 1879 and confirmed in the same Resolution of 1948,
```

Comment:
Leave them as UnitsML. This is a UnitsML rendering issue.

Reply:
Done